modular_skyrat/modules/nanotrasen_naval_command/code/outfits.dm

```diff
@@ -0,0 +1,197 @@
+/datum/outfit/centcom/naval
+
+/datum/outfit/centcom/naval/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+
+	var/obj/item/card/id/W = H.wear_id
+	W.registered_name = H.real_name
+	W.update_label()
+	W.update_icon()
+	..()
+
+/datum/outfit/centcom/naval/ensign
+	name = "Nanotrasen Naval Command - Ensign"
+
+	id = /obj/item/card/id/advanced/centcom
+	id_trim = /datum/id_trim/centcom/naval
+
+	head = /obj/item/clothing/head/caphat/naval/beret
+	ears = /obj/item/radio/headset/headset_cent/commander
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses/black
+
+	uniform = /obj/item/clothing/under/rank/centcom/naval/ensign
+
+	gloves = /obj/item/clothing/gloves/tackler/combat/insulated
+
+	l_pocket = /obj/item/melee/baton/telescopic
+
+	shoes = /obj/item/clothing/shoes/combat/swat
+
+	back = /obj/item/storage/backpack/satchel/leather
+	backpack_contents = list(/obj/item/storage/box/survival/security, /obj/item/storage/box/gunset/pdh)
+
+/datum/outfit/centcom/naval/lieutenant
+	name = "Nanotrasen Naval Command - Lieutenant"
+
+	id = /obj/item/card/id/advanced/centcom
+	id_trim = /datum/id_trim/centcom/naval/lieutenant
+
+	head = /obj/item/clothing/head/centcom_cap
+	ears = /obj/item/radio/headset/headset_cent/commander
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses/black
+
+	suit = /obj/item/clothing/suit/armor/vest/capcarapace/naval
+
+	uniform = /obj/item/clothing/under/rank/centcom/naval/commander
+
+	gloves = /obj/item/clothing/gloves/combat/naval
+
+	shoes = /obj/item/clothing/shoes/combat/swat
+
+	l_pocket = /obj/item/melee/baton/telescopic
```

Comment:
If they have the same items, can't they just go on the parent or something, so it's not repeated everywhere?